migration/sql-files/005-add-search-index.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX IF NOT EXISTS fields_title ON work_items USING GIN (to_tsvector('english', fields->>'system.title'));
```

Comment:
:) 

Thanks @aslakknutsen !

So we have to use the index:
```
CREATE INDEX IF NOT EXISTS fulltext_search_index ON work_items 
USING gin((setweight(to_tsvector('english',coalesce(fields->>'system.title','')),'B')||
    setweight(to_tsvector('english',coalesce(fields->>'system.description','')),'C')|| 
    setweight(to_tsvector('english', id::text),'A')));
```

On my system, the execution times falls from 0.6s to 0.1s with the tiny dataset.